diff -U 3 -drN a/cmake/CompizCommon.cmake b/cmake/CompizCommon.cmake
--- a/cmake/CompizCommon.cmake	2015-10-20 21:14:29.594152414 +0200
+++ b/cmake/CompizCommon.cmake	2015-10-20 21:16:40.174156934 +0200
@@ -63,7 +63,7 @@
 endif ()
 
 set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${COMMON_FLAGS}")
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMMON_FLAGS}")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMMON_FLAGS} -std=c++11")
 
 if (IS_DIRECTORY ${CMAKE_SOURCE_DIR}/.bzr)
     set(IS_BZR_REPO 1)
diff -U 3 -drN a/compizconfig/gsettings/mocks/CMakeLists.txt b/compizconfig/gsettings/mocks/CMakeLists.txt
--- a/compizconfig/gsettings/mocks/CMakeLists.txt	2015-10-20 21:17:45.544159197 +0200
+++ b/compizconfig/gsettings/mocks/CMakeLists.txt	2015-10-20 21:19:32.404162895 +0200
@@ -8,7 +8,7 @@
 			 ${CMAKE_CURRENT_SOURCE_DIR}/../gsettings_backend_shared
 			 ${GLIB_INCLUDE_DIRS})
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
     add_library (compizconfig_ccs_gsettings_wrapper_mock STATIC
 	         ${CMAKE_CURRENT_SOURCE_DIR}/ccs_gsettings_wrapper_mock.cpp
diff -U 3 -drN a/compizconfig/gsettings/tests/CMakeLists.txt b/compizconfig/gsettings/tests/CMakeLists.txt
--- a/compizconfig/gsettings/tests/CMakeLists.txt	2015-10-20 21:20:13.140830973 +0200
+++ b/compizconfig/gsettings/tests/CMakeLists.txt	2015-10-20 21:21:35.710833831 +0200
@@ -13,7 +13,7 @@
 
 pkg_check_modules (COMPIZCONFIG_TEST_GSETTINGS libcompizconfig_internal)
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
 link_directories (${CMAKE_CURRENT_BINARY_DIR}/../../libcompizconfig/tests)
 link_directories (${CMAKE_CURRENT_BINARY_DIR}/../testenvironments)
diff -U 3 -drN a/compizconfig/libcompizconfig/tests/CMakeLists.txt b/compizconfig/libcompizconfig/tests/CMakeLists.txt
--- a/compizconfig/libcompizconfig/tests/CMakeLists.txt	2015-10-20 21:22:47.394169646 +0200
+++ b/compizconfig/libcompizconfig/tests/CMakeLists.txt	2015-10-20 21:24:11.487505890 +0200
@@ -9,7 +9,7 @@
 link_directories (${CMAKE_CURRENT_BINARY_DIR}/../../tests/)
 link_directories (${CMAKE_CURRENT_BINARY_DIR}/../src/)
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
 add_executable (compizconfig_test_ccs_object
 		${CMAKE_CURRENT_SOURCE_DIR}/compizconfig_test_ccs_object.cpp)
