diff -up metacity-2.25.8/src/core/display.c.stop-spamming-xsession-errors metacity-2.25.8/src/core/display.c
--- metacity-2.25.8/src/core/display.c.stop-spamming-xsession-errors	2008-11-23 00:28:37.000000000 -0500
+++ metacity-2.25.8/src/core/display.c	2008-11-24 21:40:33.000000000 -0500
@@ -3264,10 +3264,12 @@ meta_display_begin_grab_op (MetaDisplay 
   
   if (display->grab_op != META_GRAB_OP_NONE)
     {
+#if 0
       if (window)
         meta_warning ("Attempt to perform window operation %u on window %s when operation %u on %s already in effect\n",
                       op, window->desc, display->grab_op,
                       display->grab_window ? display->grab_window->desc : "none");
+#endif
       return FALSE;
     }
 
