diff -U 3 -drN a/Makefile b/Makefile
--- a/Makefile	2015-10-04 23:04:35.000000000 +0200
+++ b/Makefile	2015-10-05 00:01:06.360473805 +0200
@@ -69,8 +69,8 @@
 build-dev: prepare $(addprefix build/lib/, ${LIBRARIES})
 
 install/lib/%:
-	mkdir -pv ${DESTDIR}${PREFIX}/share/gocode/src/${GOPKG_PREFIX}
-	cp -R ${CURDIR}/${GOPATH_DIR}/src/${GOPKG_PREFIX}/${@F} ${DESTDIR}${PREFIX}/share/gocode/src/${GOPKG_PREFIX}
+	mkdir -pv ${DESTDIR}${PREFIX}/lib/go/src/${GOPKG_PREFIX}
+	cp -R ${CURDIR}/${GOPATH_DIR}/src/${GOPKG_PREFIX}/${@F} ${DESTDIR}${PREFIX}/lib/go/src/${GOPKG_PREFIX}
 
 install-dev: build-dev ${addprefix install/lib/, ${LIBRARIES}}
 
