diff -ruN a/src/imports/publishsubscribe/qdeclarativevaluespacepublisher_p.h b/src/imports/publishsubscribe/qdeclarativevaluespacepublisher_p.h
--- a/src/imports/publishsubscribe/qdeclarativevaluespacepublisher_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/publishsubscribe/qdeclarativevaluespacepublisher_p.h	2014-12-22 02:55:43.676285921 +0100
@@ -53,7 +53,7 @@
 #ifndef QDECLARATIVEVALUESPACEPUBLISHER_H
 #define QDECLARATIVEVALUESPACEPUBLISHER_H
 
-#include <qvaluespacepublisher.h>
+#include <../../../src/publishsubscribe/qvaluespacepublisher.h>
 #include <QtCore/qstringlist.h>
 #include <QtCore/qvariant.h>
 
diff -ruN a/src/imports/publishsubscribe/qdeclarativevaluespacesubscriber_p.h b/src/imports/publishsubscribe/qdeclarativevaluespacesubscriber_p.h
--- a/src/imports/publishsubscribe/qdeclarativevaluespacesubscriber_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/publishsubscribe/qdeclarativevaluespacesubscriber_p.h	2014-12-22 02:55:34.562952272 +0100
@@ -53,7 +53,7 @@
 #ifndef QDECLARATIVEVALUESPACESUBSCRIBER_P_H
 #define QDECLARATIVEVALUESPACESUBSCRIBER_P_H
 
-#include <qvaluespacesubscriber.h>
+#include <../../../src/publishsubscribe/qvaluespacesubscriber.h>
 #include <QtCore/qstringlist.h>
 
 QT_BEGIN_NAMESPACE
diff -ruN a/src/imports/serviceframework/qdeclarativeservice.cpp b/src/imports/serviceframework/qdeclarativeservice.cpp
--- a/src/imports/serviceframework/qdeclarativeservice.cpp	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/serviceframework/qdeclarativeservice.cpp	2014-12-22 02:56:29.109620828 +0100
@@ -44,7 +44,7 @@
 #include <QQmlEngine>
 #include <QQmlInfo>
 
-#include <qservicereply.h>
+#include <../../../src/serviceframework/qservicereply.h>
 
 QT_BEGIN_NAMESPACE
 
diff -ruN a/src/imports/serviceframework/qdeclarativeserviceold_p.h b/src/imports/serviceframework/qdeclarativeserviceold_p.h
--- a/src/imports/serviceframework/qdeclarativeserviceold_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/serviceframework/qdeclarativeserviceold_p.h	2014-12-22 02:57:11.809622306 +0100
@@ -54,8 +54,8 @@
 //
 
 #include <QtCore>
-#include <qserviceinterfacedescriptor.h>
-#include <qservicemanager.h>
+#include <../../../src/serviceframework/qserviceinterfacedescriptor.h>
+#include <../../../src/serviceframework/qservicemanager.h>
 #include <qqml.h>
 #include <qqmllist.h>
 
diff -ruN a/src/imports/serviceframework/qdeclarativeservice_p.h b/src/imports/serviceframework/qdeclarativeservice_p.h
--- a/src/imports/serviceframework/qdeclarativeservice_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/serviceframework/qdeclarativeservice_p.h	2014-12-22 02:58:03.909624110 +0100
@@ -54,8 +54,8 @@
 //
 
 #include <QtCore>
-#include <qserviceinterfacedescriptor.h>
-#include <qservicemanager.h>
+#include <../../../src/serviceframework/qserviceinterfacedescriptor.h>
+#include <../../../src/serviceframework/qservicemanager.h>
 #include <qqml.h>
 #include <qqmllist.h>
 
diff -ruN a/src/imports/systeminfo/qdeclarativedeviceinfo_p.h b/src/imports/systeminfo/qdeclarativedeviceinfo_p.h
--- a/src/imports/systeminfo/qdeclarativedeviceinfo_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/systeminfo/qdeclarativedeviceinfo_p.h	2014-12-22 02:58:36.509625238 +0100
@@ -53,7 +53,7 @@
 #ifndef QDECLARATIVEDEVICEINFO_P_H
 #define QDECLARATIVEDEVICEINFO_P_H
 
-#include "qdeviceinfo.h"
+#include <../../../src/systeminfo/qdeviceinfo.h>
 
 QT_BEGIN_NAMESPACE
 
diff -ruN a/src/imports/systeminfo/qdeclarativenetworkinfo_p.h b/src/imports/systeminfo/qdeclarativenetworkinfo_p.h
--- a/src/imports/systeminfo/qdeclarativenetworkinfo_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/systeminfo/qdeclarativenetworkinfo_p.h	2014-12-22 02:59:00.659626074 +0100
@@ -53,7 +53,7 @@
 #ifndef QDECLARATIVENETWORKINFO_P_H
 #define QDECLARATIVENETWORKINFO_P_H
 
-#include <qnetworkinfo.h>
+#include <../../../src/systeminfo/qnetworkinfo.h>
 
 QT_BEGIN_NAMESPACE
 
diff -ruN a/src/imports/systeminfo/qsysteminfo.cpp b/src/imports/systeminfo/qsysteminfo.cpp
--- a/src/imports/systeminfo/qsysteminfo.cpp	2014-12-08 08:26:08.000000000 +0100
+++ b/src/imports/systeminfo/qsysteminfo.cpp	2014-12-22 03:36:16.713036811 +0100
@@ -42,10 +42,10 @@
 #include <QtQml/qqmlextensionplugin.h>
 #include <QtQml/qqml.h>
 
-#include "qbatteryinfo.h"
+#include <../../../src/systeminfo/qbatteryinfo.h>
 #include "qdeclarativedeviceinfo_p.h"
 #include "qdeclarativenetworkinfo_p.h"
-#include <qscreensaver.h>
+#include <../../../src/systeminfo/qscreensaver.h>
 
 QT_BEGIN_NAMESPACE
 
diff -ruN a/src/publishsubscribe/qvaluespace.h b/src/publishsubscribe/qvaluespace.h
--- a/src/publishsubscribe/qvaluespace.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/publishsubscribe/qvaluespace.h	2014-12-22 03:01:10.922963916 +0100
@@ -42,7 +42,7 @@
 #ifndef QVALUESPACE_H
 #define QVALUESPACE_H
 
-#include <qpublishsubscribeglobal.h>
+#include <../../../src/publishsubscribe/qpublishsubscribeglobal.h>
 
 #include <QtCore/qlist.h>
 #include <QtCore/quuid.h>
diff -ruN a/src/publishsubscribe/qvaluespacepublisher.h b/src/publishsubscribe/qvaluespacepublisher.h
--- a/src/publishsubscribe/qvaluespacepublisher.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/publishsubscribe/qvaluespacepublisher.h	2014-12-22 03:01:47.489631849 +0100
@@ -42,7 +42,7 @@
 #ifndef QVALUESPACEPUBLISHER_H
 #define QVALUESPACEPUBLISHER_H
 
-#include <qvaluespace.h>
+#include <../../../src/publishsubscribe/qvaluespace.h>
 
 #include <QtCore/qobject.h>
 #include <QtCore/qstring.h>
diff -ruN a/src/publishsubscribe/qvaluespacesubscriber.h b/src/publishsubscribe/qvaluespacesubscriber.h
--- a/src/publishsubscribe/qvaluespacesubscriber.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/publishsubscribe/qvaluespacesubscriber.h	2014-12-22 03:02:20.936299673 +0100
@@ -42,7 +42,7 @@
 #ifndef QVALUESPACESUBSCRIBER_H
 #define QVALUESPACESUBSCRIBER_H
 
-#include <qvaluespace.h>
+#include <../../../src/publishsubscribe/qvaluespace.h>
 
 #include <QtCore/qshareddata.h>
 #include <QtCore/qvariant.h>
diff -ruN a/src/serviceframework/ipc/instancemanager_p.h b/src/serviceframework/ipc/instancemanager_p.h
--- a/src/serviceframework/ipc/instancemanager_p.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/ipc/instancemanager_p.h	2014-12-22 03:03:02.679634451 +0100
@@ -56,7 +56,7 @@
 #include <qserviceframeworkglobal.h>
 #include "qremoteserviceregister.h"
 #include "qremoteserviceregisterentry_p.h"
-#include <private/qserviceclientcredentials_p.h>
+#include <qserviceclientcredentials_p.h>
 #include <QHash>
 #include <QMutexLocker>
 #include <QMetaObject>
diff -ruN a/src/serviceframework/qservicefilter.h b/src/serviceframework/qservicefilter.h
--- a/src/serviceframework/qservicefilter.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/qservicefilter.h	2014-12-22 03:03:36.496302289 +0100
@@ -42,7 +42,7 @@
 #ifndef QSERVICEFILTER_H
 #define QSERVICEFILTER_H
 
-#include "qserviceframeworkglobal.h"
+#include "../../../src/serviceframework/qserviceframeworkglobal.h"
 #include <QStringList>
 
 QT_BEGIN_NAMESPACE
diff -ruN a/src/serviceframework/qservice.h b/src/serviceframework/qservice.h
--- a/src/serviceframework/qservice.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/qservice.h	2014-12-22 03:04:14.022970255 +0100
@@ -42,7 +42,7 @@
 #ifndef Q_SERVICE_H
 #define Q_SERVICE_H
 
-#include "qserviceframeworkglobal.h"
+#include "../../../src/serviceframework/qserviceframeworkglobal.h"
 
 QT_BEGIN_NAMESPACE
 
diff -ruN a/src/serviceframework/qserviceinterfacedescriptor.h b/src/serviceframework/qserviceinterfacedescriptor.h
--- a/src/serviceframework/qserviceinterfacedescriptor.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/qserviceinterfacedescriptor.h	2014-12-22 03:04:49.096304802 +0100
@@ -42,7 +42,7 @@
 #ifndef QSERVICEINTERFACEDESCRIPTOR_H
 #define QSERVICEINTERFACEDESCRIPTOR_H
 
-#include "qserviceframeworkglobal.h"
+#include "../../../src/serviceframework/qserviceframeworkglobal.h"
 #include <QString>
 #include <QVariant>
 #include "qservice.h"
diff -ruN a/src/serviceframework/qservicemanager.h b/src/serviceframework/qservicemanager.h
--- a/src/serviceframework/qservicemanager.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/qservicemanager.h	2014-12-22 03:05:33.689639679 +0100
@@ -42,11 +42,11 @@
 #ifndef QSERVICEMANAGER_H
 #define QSERVICEMANAGER_H
 
-#include "qserviceframeworkglobal.h"
+#include "../../../src/serviceframework/qserviceframeworkglobal.h"
 
-#include "qservice.h"
-#include "qserviceinterfacedescriptor.h"
-#include "qservicefilter.h"
+#include "../../../src/serviceframework/qservice.h"
+#include "../../../src/serviceframework/qserviceinterfacedescriptor.h"
+#include "../../../src/serviceframework/qservicefilter.h"
 
 #include <QObject>
 #include <QList>
diff -ruN a/src/serviceframework/qservicereply.h b/src/serviceframework/qservicereply.h
--- a/src/serviceframework/qservicereply.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/serviceframework/qservicereply.h	2014-12-22 03:06:25.522974807 +0100
@@ -42,8 +42,8 @@
 #ifndef QSERVICEREPLY_H
 #define QSERVICEREPLY_H
 
-#include "qserviceframeworkglobal.h"
-#include "qservicemanager.h"
+#include "../../../src/serviceframework/qserviceframeworkglobal.h"
+#include "../../../src/serviceframework/qservicemanager.h"
 
 #include <QObject>
 #include <QMetaObject>
diff -ruN a/src/systeminfo/linux/qdeviceinfo_linux.cpp b/src/systeminfo/linux/qdeviceinfo_linux.cpp
--- a/src/systeminfo/linux/qdeviceinfo_linux.cpp	2014-12-08 08:26:08.000000000 +0100
+++ b/src/systeminfo/linux/qdeviceinfo_linux.cpp	2014-12-22 03:26:52.566350616 +0100
@@ -50,7 +50,7 @@
 #endif // QT_USE_SSU
 
 #include "qscreensaver_linux_p.h"
-#include <QNetworkInfo>
+#include <../../../src/systeminfo/qnetworkinfo.h>
 
 #include <QtCore/qdir.h>
 #include <QtCore/qmetaobject.h>
diff -ruN a/src/systeminfo/qbatteryinfo.h b/src/systeminfo/qbatteryinfo.h
--- a/src/systeminfo/qbatteryinfo.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/systeminfo/qbatteryinfo.h	2014-12-22 03:06:58.432975946 +0100
@@ -43,7 +43,7 @@
 #ifndef QBATTERYINFO_H
 #define QBATTERYINFO_H
 
-#include <QtSystemInfo/qsysteminfoglobal.h>
+#include <../../../src/systeminfo/qsysteminfoglobal.h>
 #include <QtCore/qobject.h>
 
 QT_BEGIN_NAMESPACE
diff -ruN a/src/systeminfo/qdeviceinfo.h b/src/systeminfo/qdeviceinfo.h
--- a/src/systeminfo/qdeviceinfo.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/systeminfo/qdeviceinfo.h	2014-12-22 03:07:21.402976741 +0100
@@ -42,7 +42,7 @@
 #ifndef QDEVICEINFO_H
 #define QDEVICEINFO_H
 
-#include <QtSystemInfo/qsysteminfoglobal.h>
+#include <../../../src/systeminfo/qsysteminfoglobal.h>
 #include <QtCore/qobject.h>
 
 QT_BEGIN_NAMESPACE
diff -ruN a/src/systeminfo/qnetworkinfo.h b/src/systeminfo/qnetworkinfo.h
--- a/src/systeminfo/qnetworkinfo.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/systeminfo/qnetworkinfo.h	2014-12-22 03:07:46.096310930 +0100
@@ -42,7 +42,7 @@
 #ifndef QNETWORKINFO_H
 #define QNETWORKINFO_H
 
-#include <QtSystemInfo/qsysteminfoglobal.h>
+#include <../../../src/systeminfo/qsysteminfoglobal.h>
 #include <QtCore/qobject.h>
 #include <QtNetwork/qnetworkinterface.h>
 
diff -ruN a/src/systeminfo/qscreensaver.h b/src/systeminfo/qscreensaver.h
--- a/src/systeminfo/qscreensaver.h	2014-12-08 08:26:08.000000000 +0100
+++ b/src/systeminfo/qscreensaver.h	2014-12-22 03:08:14.039645230 +0100
@@ -42,7 +42,7 @@
 #ifndef QSCREENSAVER_H
 #define QSCREENSAVER_H
 
-#include <QtSystemInfo/qsysteminfoglobal.h>
+#include <../../../src/systeminfo/qsysteminfoglobal.h>
 #include <QtCore/qobject.h>
 
 QT_BEGIN_NAMESPACE
