diff --git a/bin/dde-session-daemon/daemon.go b/bin/dde-session-daemon/daemon.go
index f7abf89..fdbd113 100644
--- a/bin/dde-session-daemon/daemon.go
+++ b/bin/dde-session-daemon/daemon.go
@@ -10,7 +10,8 @@ import _ "pkg.linuxdeepin.com/dde-daemon/audio"
 import _ "pkg.linuxdeepin.com/dde-daemon/appearance"
 
 import _ "pkg.linuxdeepin.com/dde-daemon/clipboard"
-import _ "pkg.linuxdeepin.com/dde-daemon/datetime"
+
+// import _ "pkg.linuxdeepin.com/dde-daemon/datetime"
 import _ "pkg.linuxdeepin.com/dde-daemon/mime"
 
 import _ "pkg.linuxdeepin.com/dde-daemon/screenedge"
