diff -ruN a/data/lightdm.conf b/data/lightdm.conf
--- a/data/lightdm.conf	2014-05-26 02:18:25.695909663 +0200
+++ b/data/lightdm.conf	2014-05-26 02:25:00.232589987 +0200
@@ -19,12 +19,12 @@
 #start-default-seat=true
 #greeter-user=lightdm
 #minimum-display-number=0
-#minimum-vt=7
+#minimum-vt=1
 #lock-memory=true
 #user-authority-in-system-dir=false
 #guest-account-script=guest-account
 #log-directory=/var/log/lightdm
-#run-directory=/var/run/lightdm
+#run-directory=/run/lightdm
 #cache-directory=/var/cache/lightdm
 #sessions-directory=/usr/share/lightdm/sessions:/usr/share/xsessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
@@ -85,15 +85,15 @@
 #xdmcp-key=
 #unity-compositor-command=unity-system-compositor
 #unity-compositor-timeout=60
-#greeter-session=example-gtk-gnome
+greeter-session=lightdm-deepin-greeter
 #greeter-hide-users=false
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
 #greeter-show-remote-login=true
-#user-session=default
+user-session=deepin
 #allow-guest=true
 #guest-session=
-#session-wrapper=lightdm-session
+session-wrapper=/etc/lightdm/Xsession
 #greeter-wrapper=
 #guest-wrapper=
 #display-setup-script=
@@ -107,6 +107,7 @@
 #autologin-in-background=false
 #autologin-session=UNIMPLEMENTED
 #exit-on-failure=false
+pam-service=lightdm-autologin
 
 #
 # Seat configuration
diff -ruN a/data/users.conf b/data/users.conf
--- a/data/users.conf	2014-05-26 02:21:14.402582170 +0200
+++ b/data/users.conf	2014-05-26 02:22:18.779251064 +0200
@@ -9,6 +9,6 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserList]
-minimum-uid=500
+minimum-uid=1000
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/bin/false /sbin/nologin
