--- Makefile.old	2014-08-25 12:19:42.000000000 +0200
+++ Makefile	2014-08-25 12:21:56.858575247 +0200
@@ -33,18 +33,18 @@
 	env GOPATH="${CURDIR}/${GOPATH_DIR}:${GOPATH}" go install ${GOPKG_PREFIX}
 
 install-prepare:
-	mkdir -p ${DESTDIR}${PREFIX}/share/go/src/pkg/${GOPKG_PREFIX}
+	mkdir -p ${DESTDIR}${PREFIX}/lib/go/src/pkg/${GOPKG_PREFIX}
 	mkdir -p ${DESTDIR}${PREFIX}/lib/go/pkg
 
 install/src/%:
-	cp -r ${CURDIR}/${@F} ${DESTDIR}${PREFIX}/share/go/src/pkg/${GOPKG_PREFIX}/
+	cp -r ${CURDIR}/${@F} ${DESTDIR}${PREFIX}/lib/go/src/pkg/${GOPKG_PREFIX}/
 
 install: build install-prepare $(addprefix install/src/, ${LIBS})
-	cp -r ${CURDIR}/*.go ${DESTDIR}${PREFIX}/share/go/src/pkg/${GOPKG_PREFIX}/
+	cp -r ${CURDIR}/*.go ${DESTDIR}${PREFIX}/lib/go/src/pkg/${GOPKG_PREFIX}/
 	cp -r ${CURDIR}/${GOPATH_DIR}/pkg/* ${DESTDIR}${PREFIX}/lib/go/pkg/
 
 uninstall:
-	rm -rvf ${DESTDIR}${PREFIX}/share/go/src/pkg/${GOPKG_PREFIX}
+	rm -rvf ${DESTDIR}${PREFIX}/lib/go/src/pkg/${GOPKG_PREFIX}
 	rm -rvf ${DESTDIR}${PREFIX}/lib/go/pkg/${GOOS}_${GOARCH}/${GOPKG_PREFIX}
 
 clean:
